examples/md-flexible/src/domainDecomposition/RegularGridDecomposition.cpp

```diff
@@ -252,8 +258,11 @@ void RegularGridDecomposition::exchangeMigratingParticles(AutoPasType &autoPasCo
   }
 }
 
-void RegularGridDecomposition::reflectParticlesAtBoundaries(AutoPasType &autoPasContainer) {
+void RegularGridDecomposition::reflectParticlesAtBoundaries(AutoPasType &autoPasContainer,
+                                                            ParticlePropertiesLibraryType &PPL) {
```

Comment:
```suggestion
void RegularGridDecomposition::reflectParticlesAtBoundaries(AutoPasType &autoPasContainer,
                                                            ParticlePropertiesLibraryType &particlePropertiesLib) {
```
(will require a few more renames)